### PR TITLE
Fix/slot cleanup

### PR DESCRIPTION
**Pull request by TrevorJTClarke**

### CODE REVIEW

**Comment by TrevorJTClarke** [comment]
fixes #21

**Comment by TrevorJTClarke** [comment]
Need to clean up the commented code, then this is ready

**Comment by TrevorJTClarke** [comment]
@mikedotexe this is ready for your review pwease :D

File: manager/src/lib.rs [hunk 1]
```diff
@@ -247,20 +228,17 @@ impl CronManager {
...
             (ret, U128::from(current_slot))
         } else {
             (vec![], U128::from(current_slot))
         }
     }
 
-    /// Most useful for debugging at this point.
+    /// Returns task data
+    /// Used by the frontend for viewing tasks
+    /// REF: https://docs.near.org/docs/concepts/data-storage#gas-consumption-examples-1
```
Comment (mikedotexe) [inline manager/src/lib.rs:241]: 💥

File: manager/src/lib.rs [hunk 2]
```diff
@@ -517,58 +529,35 @@ impl CronManager {
...
         }
 
+        // Clean up slot if no more data
+        if slot_data.len() < 1 { self.slots.remove(&slot_ballpark.unwrap()); }
+
         let mut task = self.tasks.get(&hash).expect("No task found by hash");
         log!("Found Task {:?}", &task);
 
-        // Check that task status is ready or completed
-        assert!(
```
Comment (mikedotexe) [inline manager/src/lib.rs:530]: yep yep yep

**Review by mikedotexe — APPROVED** [review]

**Pull request merged**